daemon/src/wire.rs

```diff
@@ -174,6 +175,10 @@ pub enum MakerToTaker {
         oracle_event_id: BitMexPriceEventId,
         tx_fee_rate: TxFeeRate,
         funding_rate: FundingRate,
+        /// Fee to be paid for the rollover.
+        ///
+        /// The maker comes up with this amount so that both parties are on the same page
+        complete_fee: FeeFlow,
```

Comment:
🛠 I don't think it's a good idea to use the `FeeFlow` directly here. I think it would be cleaner to convert into a struct/enum that is used over the wire only, so we can change the `FeeFlow` without breaking the wire messages.

Reply:
Happy to introduce a different struct for that.

Reply:
Actually, this is not easily to be changed. `FeeFlow` is used all the way down to the `payout_curve`

Reply:
It's not easy to introduce a different type here given how `FeeFlow` is implemented. I've opted for just converting it into an enum owned by wire to achieve what you were proposing. 

[`93fdaa2` (#1981)](https://github.com/itchysats/itchysats/pull/1981/commits/93fdaa2a65b118babeb8d5980bb42ec1392745ae)